src/components/grid/index.tsx

```diff
@@ -1,19 +1,13 @@
-import React from 'react';
+export { type GridProps, type GridItemProps } from '@chakra-ui/react';
 import {
   Grid as ChakraGrid,
-  ComponentWithAs,
-  forwardRef,
-  type GridProps,
+  GridItem as ChakraGridItem,
 } from '@chakra-ui/react';
 
-const Grid: ComponentWithAs<'div', GridProps> = forwardRef<GridProps, 'div'>(
-  (props, ref) => {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    return <ChakraGrid {...(props as GridProps)} ref={ref} />;
-  },
-);
-
+const Grid = ChakraGrid;
 Grid.displayName = 'Grid';
```

Comment:
Here's a gotcha. Chakra components rely a lot on `React.forwardRef` and named function as an argument to it for display name generation. This works fine with browser extensions like react devtools, but you'll see
```
<React.ForwardRef>...</React.ForwardRef>
```
in storybook.
Adding display name directly remediates that

Reply:
In case someone else is looking where in storybook you seed that. The comment refers to the generated code snippets.
![Screenshot 2024-11-22 at 11 15 00](https://github.com/user-attachments/assets/683d0493-40d8-4c1e-915a-c8f532aa3692)
![Screenshot 2024-11-22 at 11 14 48](https://github.com/user-attachments/assets/4990c309-71bc-4a12-83d4-ecb5d71364cb)